frontend/src/services/graphQL.ts

```diff
@@ -32,14 +35,57 @@ export async function graphQLRequest(query: String, variables: ILookup<any> = {}
   return await axios.request(request)
 }
 
+export async function graphQLRequestWithErrorHandling(query: String, variables: ILookup<any> = {}, parse : any = undefined) {
```

Comment:
```suggestion
export async function graphQLRequestWithErrorHandling(query: String, variables: ILookup<any> = {}, parse?: Boolean  = undefined) {
```

Reply:
parse will be a function.

Reply:
I think this should work. I try to avoid `any`
```suggestion
export async function graphQLRequestWithErrorHandling(query: String, variables: ILookup<any> = {}, parse?: () => void = undefined) {
```